config/config.template.ini

```diff
@@ -115,6 +115,10 @@ netIdent_ric = 0174760, 1398098
 # you can hold one entry per netIdent_ric [0] or the whole history [1]
 netIdent_history = 0
 
+# If a RIC contains coordinates, specify the regex used to decode them
+geo_format = #C(\d{2})(\d{5}),(\d{2})(\d{5})#

```

Comment:
Sollte man hier ein Flag zum an-/ausschalten implementieren?

Reply:
Hatte ich überlegt, perspektivisch wollte ich jedoch alle POC-Nachrichten über einen Geodecoder jagen.

Reply:
Versteh ich jetzt nicht ganz.
Was ich meinte ist, ein Flag in der Config mit dem man das "Regex'en" der Geo-Daten komplett ausschalten kann.
Bzw. wie sieht das aktuell aus? Wenn keine Regex String angeben ist? Führt das zu einer Exception, oder ist das `has_Geo`-Flag dann sowieso false?